modules/core/src/main/java/org/apache/ignite/internal/processors/cluster/GridClusterStateProcessor.java

```diff
@@ -1045,6 +1025,7 @@ else if (ClusterState.active(state)) {
     /** */
     private IgniteInternalFuture<?> changeGlobalState0(
         ClusterState state,
+        boolean force,
```

Comment:
We should rename this flag to `forceDeactivation` because its value applies to the deactivation command only.

Reply:
Fixed.